### ISSUE TITLE
Problem with hover over on menu items and UI buttons.

### ISSUE THREAD

**Issue by mbrooks777** [kbuffington/Georgia#43]
I have installed your Georgia theme after seeing it mentioned on Reddit and I really enjoy it.
However, I have one issue that I can't seem to fix and I have looked at all your faq locations and could not find the solution.
Whenever I hover over the menu tabs or the transport buttons or the option buttons, all the icons have a white background that I can't seem to get rid off. 
![help1](https://user-images.githubusercontent.com/67404834/85656240-8dd67c80-b67e-11ea-8607-a6db2574832b.png)

**Comment by mbrooks777** [comment]
Hi , this is happening on Windows 10 and a vanilla install of foobar 1.5.4 and I installed all the components you suggested from Georgia 1.1.8 including installing the custom fonts and I am using foo_ui_columns 1.4.1 and foo_jscript_panel 2.3.6.1

**Comment by kbuffington** [comment]
That's.... weird. It appears that something is wrong with GDI on your machine. The white boxes are the bounding boxes of where a repaint notification should occur. It's as if the GDI frame buffer can't see the existing background. I'm on Windows 10 and Foobar 1.5.4 and have never seen this before. I'm assuming you tried restarting foobar. Might see if restarting Windows makes the problem correct itself.

**Comment by mbrooks777** [comment]
Thank you for the reply! After trying every option I could think of, including turning the 4k option in your skin on and off, I remembered that I had changed the scaling in property settings for foobar2k. It seems that if you use the properties - compatibility - change high DPI settings button - high DPI scaling override - setting to system (enhanced) causes this problem. I turned this setting off and now everything works!
